templates/account/invoices/index.html

```diff
@@ -0,0 +1,70 @@
+{% extends "advantage/base_advantage.html" %}
+
+{% block title %}Invoices{% endblock %}
+{% block meta_description %}Ubuntu Advantage for Infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry, with OpenStack support, Kubernetes support included, and Livepatch, Landscape and Extended Security Maintenance to address security and compliance concerns.{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-12">
+      <h1>Invoices</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-3">
+      <select name="marketplace" id="marketplace-dropdown">
+        <option value="">All invoices</option>
+        <option value="canonical-ua" {% if marketplace == 'canonical-ua' %}selected{% endif %}>Canonical UA</option>
```

Comment:
Question for UX. But wouldn't "Ubuntu Advantage" be better here inside of Canonical UA?

Reply:
@wgx @lechsandecki thoughts?

We call the name of the service inside the table also `Canonical UA`.
![image](https://user-images.githubusercontent.com/6387619/126626738-ed2877be-b93d-4158-8791-dde0c888ea90.png)

Reply:
@albertkol this is fine for now, but @lechsandecki may want to change these